blocks/editable/index.js

```diff
@@ -662,3 +662,24 @@ export default class Editable extends Component {
 Editable.contextTypes = {
 	onUndo: noop,
 };
+
+Editable.Value = ( { value } ) => valueToReact( value );
+
+function valueToReact( value ) {
+	if ( ! Array.isArray( value ) ) {
+		return value;
+	}
+
+	const result = value.map( ( element, i ) => {
+		if ( ! Array.isArray( element ) ) {
+			return element;
+		}
+
+		const [ type, props, children ] = element;
+
+		return createElement( type, { ...props, key: i }, valueToReact( children ) );
+	} );
+
+	// We are still using React 15, so can't return array or string directly...
```

Comment:
Only in test environment are we using React 15.

Reply:
Hopefully not that long: https://github.com/WordPress/gutenberg/pull/3079 :)